docs/administration/spaces/index.md

```diff
@@ -4,76 +4,134 @@ description: Spaces setup and configuration.
 
 You can modify a space by navigating to {{Configuration,Spaces}} and selecting the space you want to modify. Expand the field you would like to change and click **SAVE** to save your changes.
 
+Here you can rename a space, give it a new logo, modify the space managers, or even stop the spaces task queue from processing.
+
+![Modify a space](modify-space.png)
+
 ### Delete a Space
 
-You can delete spaces when you are the **Space Owner**. Deleting a space cannot be undone, and the space and all of its contents, including projects, environments, releases, and deployment history will be deleted.
+You can delete spaces when you are the **Space manager**. Deleting a space cannot be undone, and the space and all of its contents, including projects, environments, releases, and deployment history will be deleted.
 
-1. Navigate to {{Configuration,Spaces}} and selecT the space you want to delete.
+1. Navigate to {{Configuration,Spaces}} and select the space you want to delete.
 1. Expand the **Task Queue Status** section and select the Stop task queue checkbox, and click **SAVE**.
 1. Click the overflow button and select **Delete**.
 1. Enter the name of the space and click **DELETE**.
 
+## Change the Default Space {#change-the-default-space}
+
+The **Default space** is provided to existing installations as a mechanism to ensure that the instance operates in much the same way as it did prior to upgrading to a 'spaces enabled' build. Enabled by default, it's primary function is to provide an initial space for any existing artifacts, including our built in data at installation time, and effectively hides the existence of spaces until you are ready to start using them.
+
+It is possible to change the default space:
+
+> Note: that the following carries some minor downtime for any automation that relies on the default space being available
+ 
+1. Navigate to {{Configuration,Spaces}} and [disable the default space](#disable-the-default-space)
+2. Return to {{Configuration,Spaces}} and select the space that you wish to nominate as the default space
+3. Click the overflow button and select **Enable the default space**.
+
 ## Disable the Default Space {#disable-the-default-space}
 
-You can disable the default space. <!-- content explaining the ramifications goes here -->
+For organizations that are new to Octopus, especially those that make heavy use of spaces, a default space is not necessarily required and therefore we have made it possible to remove the default space entirely. However, this comes with some considerations that should be weighed carefully against the needs of your organization.
+
+In addition to providing a home for existing resources, the default space allows any existing API calls that do not explicitly set a `Space Identifier` in the route, to be routed to the default space. For example, in the case where the default space ID is `Spaces-1` then the route `/api/projects/my-project` is equivalent to `/api/Spaces-1/projects/my-project`.
```

Comment:
As I mentioned in another comment, I think this paragraph should be up in the `Change the Default Space` section